src/components/Layer2ProductCard.tsx

```diff
@@ -110,21 +41,61 @@ const Layer2ProductCard: React.FC<IProps> = ({
   const intl = useIntl()
 
   return (
-    <Card>
-      <ImageWrapper background={background}>
-        <Image image={image} alt={alt} objectFit="contain" />
-      </ImageWrapper>
-      <Content className="hover">
-        <div>
-          <Title>{name}</Title>
-          {children && <Children>{children}</Children>}
-          <Description>{description}</Description>
+    <Flex
+      color="text"
+      boxShadow={"0px 14px 66px rgba(0, 0, 0, 0.07)"}
+      direction="column"
+      justify="space-between"
+      bg="searchBackground"
+      borderRadius={1}
+      border={"1px solid lightBorder"}
+      textDecoration="none"
+      padding={2}
+      _hover={{ transition: "transform 0.1s", transform: "scale(1.02)" }}
+    >
+      <Flex
+        bg={background}
+        direction="row"
+        justify="center"
+        align="center"
+        boxShadow="inset 0px -1px 0px rgba(0, 0, 0, 0.1)"
+        minH="200px"
+      >
+        <Image
+          as={GatsbyImage}
+          image={image}
+          alt={alt}
+          objectFit="contain"
+          alignSelf="center"
+          maxW={{ base: "311px", sm: "372px" }}
+          maxH={"257px"}
+        />
+      </Flex>
+      <Flex
+        p={6}
+        textAlign="left"
+        h="100%"
+        direction="column"
+        className="hover"
```

Comment:
Does this do something? if not then remove it

Reply:
Hi @pabloped i removed the `textAlign` as it does nothing but kept the `h` and `direction` bc without it the styling with the links gets messed up. Here's a photo for reference 

![image](https://user-images.githubusercontent.com/61770010/193176863-d3818f8d-4716-4940-a64d-78a0a16eed9f.png)

Reply:
Apologies, you are indeed correct its not needed here